### DISCUSSION TITLE
Online Demo

### DISCUSSION THREAD

**Discussion by Amaanuddin** [AnasAito/SkillNER#55] [Q&A] [answered]
Hi, is there any live website/playground for this? I want to try this API.
if there is any website for this can some send me the link.

**Comment by Badr-MOUFAD** [comment]
Yes, we do have a demo App to try the API.
Follow this link: https://anasaito-skillner-demo-index-4fiwi3.streamlit.app/

**Reply by Amaanuddin** [reply]
It's not working 
![Screenshot_2022-11-24-23-48-29-92_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/29299860/203846934-6b9d989c-9907-4c00-8ada-d1586beb8f3c.jpg)

**Comment by Badr-MOUFAD** [comment, marked as answer]
@Amaanuddin, it should be working now on, can you check?

**Reply by Amaanuddin** [reply]
Yes, it's working now. Thanks for updating.

**Comment by AnAnalogGuy** [comment]
https://anasaito-skillner-demo-index-4fiwi3.streamlit.app/ is not running anymore.
